app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  after_save :update_user_posts_counter
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] As mentioned previously it'd be better to make the `limit` value of the `recent_comments` method internal to the method ensuring it doesn't get overridden when invoking the method as it should always return the 5 most recent comments for a given post.